packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/quanticRefineModalContent.js

```diff
@@ -32,7 +31,7 @@ import {LightningElement, api} from 'lwc';
  * @category Search
  * @category Insight Panel
  * @example
- * <c-quantic-refine-modal-content engine-id={engineId} hide-sort></c-quantic-refine-modal-content>
+ * <c-quantic-refine-modal-content engine-id={engineId} hide-sort disable-facet-ordering={disableFacetOrdering}></c-quantic-refine-modal-content>
```

Comment:
```suggestion
 * <c-quantic-refine-modal-content engine-id={engineId} hide-sort disable-facet-ordering></c-quantic-refine-modal-content>
```

This should be what's valid in the html.

Reply:
Just having the attribute there on the HTML is equivalent to writing `disable-facet-ordering="true"`.